src/libs/ReceiptUtils.ts

```diff
@@ -65,7 +66,7 @@ function getThumbnailAndImageURIs(transaction: Transaction, receiptPath: string
         image = ReceiptSVG;
     }
 
-    return {thumbnail: null, image};
+    return {thumbnail: image, image: path, isLocalFile: true};
```

Comment:
You forgot to cover the case of local files which caused a bug. More details here: https://github.com/Expensify/App/issues/33474